src/pages/CartPage/CartPage.jsx

```diff
@@ -1,7 +1,70 @@
 import React from "react";
+import "./CartPage.css";
+import Nav from "../../components/Nav/Nav";
```

Comment:
you can create `index.js` file for folders and export components from there.

Reply:
Sure. After making all i will move it to index.js file